### ISSUE TITLE
File & Folder Convention

### ISSUE THREAD

**Issue by byronmejia** [codenetwork/CodeNetwork.co#11]
Okay, this one is serious. So, I'm struggling hard to understand where everything should go here.

Here's an example of a layout that could work well.
![image](https://cloud.githubusercontent.com/assets/11262664/13797755/be51ed38-eb5b-11e5-918e-e74c20a0f4fe.png)

Another suggestion would be to possibly cut out angular for now. I feel that it's added a layer of complication that I'm not ready for yet.

**Comment by trjstewart** [comment]
Yeah I've been considering dropping angular for now in favor of making the project easier to contribute to. I'll have a chat to you about it today and run you through the folder structure and we can make the decision from there.

**Comment by AdamHibble** [comment]
VanillaJS?
